Doc/using/windows.rst

```diff
@@ -625,17 +630,21 @@ Customizing default Python versions
 In some cases, a version qualifier can be included in a command to dictate
 which version of Python will be used by the command. A version qualifier
 starts with a major version number and can optionally be followed by a period
-('.') and a minor version specifier. If the minor qualifier is specified, it
-may optionally be followed by "-32" to indicate the 32-bit implementation of
-that version be used.
+('.') and a minor version specifier. Furthermore it is possible to specifiy
+if a 32 or 64 bit implementation shall be enforced by adding "-32" or "-64".
+Please notice that "-64" (in contrary to "-32") and the variant "major-bits"
+(without specifiying a minor) are available only since Python Launcher 3.7
+or newer.
```

Comment:
Again, I don't think you need to mention the Python Launcher here.  Also, you would only need the versionadded in one place.